app/controllers/application_controller.rb

```diff
@@ -10,13 +10,14 @@ def current_user
     @current_user ||= User.find(session[:user_id]) if session[:user_id]
   end
 
-  def find_bookmark(id)
-    current_user.user_videos.find_by(video_id: id)
-  end
+  # def find_bookmark(id)
+  #   binding.pry
+  #   current_user.user_videos.find_by(video_id: id)
+  # end
```

Comment:
more tech debt that is not implemented

Reply:
great! I'll try to tally up a line count of how many methods and routes weren't used to put in our README regarding cleaning up tech debt in the existing application.